hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/TestBlockDeletingService.java

```diff
@@ -70,6 +72,7 @@
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
```

Comment:
This test is still in JUnit 4, `org.junit.Assert` would be more appropriate (which is already imported).

Reply:
Thx for pointing out this ~ updated